examples/extension/README.md

```diff
@@ -34,13 +34,19 @@ For more information, see the `extendedAgent` task in [build.gradle](build.gradl
 
 ## Extensions examples
 
+* Custom `AutoConfigurationCustomizer`: [DemoAutoConfigurationCustomizerProvider](src/main/java/com/example/javaagent/DemoAutoConfigurationCustomizerProvider.java)
 * Custom `IdGenerator`: [DemoIdGenerator](src/main/java/com/example/javaagent/DemoIdGenerator.java)
 * Custom `TextMapPropagator`: [DemoPropagator](src/main/java/com/example/javaagent/DemoPropagator.java)
 * Custom `Sampler`: [DemoSampler](src/main/java/com/example/javaagent/DemoSampler.java)
 * Custom `SpanProcessor`: [DemoSpanProcessor](src/main/java/com/example/javaagent/DemoSpanProcessor.java)
 * Custom `SpanExporter`: [DemoSpanExporter](src/main/java/com/example/javaagent/DemoSpanExporter.java)
 * Additional instrumentation: [DemoServlet3InstrumentationModule](src/main/java/com/example/javaagent/instrumentation/DemoServlet3InstrumentationModule.java)
 
+To apply the above extension classes to the Java Agent, include an `AutoConfigurationCustomizer` in
+your extension.
+See [DemoAutoConfigurationCustomizerProvider](src/main/java/com/example/javaagent/DemoAutoConfigurationCustomizerProvider.java)
+for an example.
```

Comment:
That's still not strictly true though: you can add an extra instrumentation (or a propagator, see `DemoPropagatorProvider`) without having an `AutoConfigurationCustomizer` in your extension. WDYT about editing this entire block so that it lists the SPI implementations first, and then maybe some of the things that you can use them for?

Reply:
Ok, I've updated the explanation.  Better?